src-docs/src/views/super_select/super_select_example.js

```diff
@@ -54,6 +59,23 @@ const superSelectStatesSnippet = `<EuiSuperSelect
 
 export const SuperSelectExample = {
   title: 'Super select',
+  intro: (
+    <EuiCallOut
+      iconType="questionInCircle"
+      title="Learn more about EUI selection components"
```

Comment:
I liked the question/answer format of the previous callouts - maybe we can tweak this one callout to use the old copy?
```suggestion
      title="Not sure which selection component to use?"
```